.github/workflows/master-e2e.yaml

```diff
@@ -241,10 +244,15 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 16
+      - name: Install proxy
+        if: ${{ inputs.proxy == 'rancher' }}
+        run: |
+          docker run -d --name squid_proxy -v $(pwd)/.github/scripts/squid.conf:/etc/squid/squid.conf -p 3128:3128 wernight/squid
```

Comment:
please use some less obscure image and not this which is 7y old. I recommend `ubuntu/squid`

Reply:
Okay, I will explore and update image accordingly.